src/scss/pages/_a-z-index.scss

```diff
@@ -0,0 +1,50 @@
+.a-z-index {
+
+  .resource-search-label {
+    display: block;
+    position: relative;
+
+    h2 {
+      font-size: 1.25em;
+      margin-bottom: 1.3rem;
+    }
+
+    &::after {
+      background-color: #fff;
+      color: #ccc;
+      content: '\f002';
+      display: block;
+      font-family: FontAwesome;
+      font-size: 34px;
+      font-weight: 400;
+      height: 34px;
+      line-height: 34px;
+      padding-left: 5px;
+      padding-right: 10px;
+      position: absolute;
+      right: 10px;
+      top: 58px;
+      z-index: 2;
+    }
+  }
+
+  .resource-search-heading {
+    font-family: $font-family-slab-serif;
+  }
+
+  .back-to-top a {
+    color: $gray-darker;
+    font-size: .9em;
+    text-transform: uppercase;
+
+    &::before {
+      content: '\f062';
+      font-family: FontAwesome;
+      margin-right: .3em;
+    }
+  }
+}
+
+.list-navigation li {
```

Comment:
Consider moving this within the .a-z-index selector to avoid overriding this rule globally